feflow/protocols/nonequilibrium_cycling.py

```diff
@@ -325,9 +325,14 @@ def _execute(self, ctx, *, state_a, state_b, mapping, settings, **inputs):
             openmm.LocalEnergyMinimizer.minimize(context)
 
             # SERIALIZE SYSTEM, STATE, INTEGRATOR
+            thermodynamic_settings = settings.thermo_settings
+            temperature = to_openmm(thermodynamic_settings.temperature)
```

Comment:
Possibly worth adding a comment on why this is necessary? Ideally a test too, but at least the former so we don't accidentally do a regression if we refactor stuff.

Reply:
Done! Added comments indicating the importance of these changes to Folding@Home use!